pyccl/tracers.py

```diff
@@ -484,8 +484,11 @@ class WeakLensingTracer(Tracer):
             (z, A_IA(z)) giving the intrinsic alignment amplitude A_IA(z).
             If `None`, the tracer is assumped to not have intrinsic
             alignments. Defaults to None.
+        ia_renorm (bool): set to True to use the conventional IA normalization.
+            set to False to use the raw input amplitude, which will usually be 1
+            for use with PT IA modeling. Defaults to True.
     """
-    def __init__(self, cosmo, dndz, has_shear=True, ia_bias=None):
+    def __init__(self, cosmo, dndz, has_shear=True, ia_bias=None, ia_renorm=True):
```

Comment:
Can we be more explicit in the choice of keyword? For example, set_ia_norm_aia, or similar? Am I right to assume this normalization option also works for NLA?

Reply:
I have updated this to `fid_ia_norm`. This option doesn't know anything about which power spectrum is being used, so it would also apply to NLA if you wanted to specify the normalization yourself. But it is necessary for nonlinear IA. I assume that all of the IA normalization is being done here at the python level.

Reply:
updated again to `use_a_ia` to distinguish between specifying `a` and `c`